scripts/services/dependencies.ts

```diff
@@ -66,6 +66,7 @@ const proxyChecks: Array<Check> = [
     "getnetworkinfo",
     "getrawmempool",
     "getrawtransaction",
+    "sendrawtransaction",
```

Comment:
This is actually required to send a transaction!  I'm not sure how anybody using btc-rpc-proxy was able to send from wallets before.  I confirmed that missing this was blocking electrum and green from being able to send transactions.

Reply:
Yep, amazing that I missed this, thanks for catching!